core/config/engine.cpp

```diff
@@ -248,6 +248,9 @@ String Engine::get_architecture_name() const {
 	return "ppc";
 #endif
 
+#elif defined(__loongarch__)
+	return "loongarch64";
```

Comment:
https://loongson.github.io/LoongArch-Documentation/LoongArch-toolchain-conventions-EN.html

According to this page, `__loongarch64` is a more specific check than `__loongarch__`, which we can check for to return `"loongarch64"`, and otherwise fall back to a more generic bitless name (like the rv64 code above), ensuring that in case somebody tries to compile on an 32-bit or 128-bit version of the architecture then Godot doesn't still claim to be the 64-bit version (or it can just not compile in those cases).

Reply:
I decide only do more specific check without fallback to generic name. there is no 32-bit LoongArch desktop CPU yet. I think it's better to notify future developers we didn't actually test it by explicitly only support 64-bit LoongArch CPU.